dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelOpenApiExtensions.cs

```diff
@@ -21,6 +21,7 @@
 using Microsoft.SemanticKernel.Skills.OpenAPI.Model;
 using Microsoft.SemanticKernel.Skills.OpenAPI.OpenApi;
 using Microsoft.SemanticKernel.Skills.OpenAPI.Skills;
+using Newtonsoft.Json;
```

Comment:
Are we still using Newtonsoft.Json, or trying to shift over to System.Text.Json?

Reply:
We should use System.Text.Json wherever possible. As I recall, the nuget package we call to parse OpenAPI docs currently uses newtonsoft so, unfortunately, this package brings it in.